crates/bdk/src/wallet/mod.rs

```diff
@@ -1662,10 +1664,12 @@ impl Wallet {
                     change_fee,
                 } = excess
                 {
-                    return Err(CreateTxError::InsufficientFunds {
-                        needed: *dust_threshold,
-                        available: remaining_amount.saturating_sub(*change_fee),
-                    });
+                    return Err(CreateTxError::CoinSelection(
+                        Error::InsufficientFunds {
```

Comment:
@e1a0a0ea You could probably use a qualified version: `coin_selection::Error::InsufficientFunds` instead 🤔

Reply:
Or just import the `InsufficientFunds` variant itself.

Reply:
I'm not a fan of importing the enum variant (can become confusing imo). I prefer the more verbose, but clearer option.